src/store.js

```diff
@@ -0,0 +1,10 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+
+Vue.use(Vuex)
```

Comment:
Are there any drawbacks to installing the Vuex plugin here when we currently aren't making use of it?

Reply:
There shouldn't be. It's nice having Vuex ready to go since we should be making use of it very soon (I'm hoping to begin using it during the dashboard redesign). Still, if you think we should remove it until then, I can comment it out.

Reply:
Yeah, makes sense to leave it in!